### PR TITLE
Content Revamp

### PR DESCRIPTION
**Pull request by daltonfury42**
I redid most of the home page content.

### CODE REVIEW

File: simplq/src/components/pages/Home/LandingPage.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import styles from './Home.module.scss';
 import MyQueues from './MyQueues';
 
 export default () => {
-  let subtitle = 'A long overdue alternative to physical queues';
+  let subtitle = 'Give Your Customers a Delightful Customer Waiting Experience.';
   const { user, isAuthenticated } = useAuth0();
```
Comment (maaverik) [inline simplq/src/components/pages/Home/LandingPage.jsx:10]: Let's remove the second cutomer? - Give Your Customers a Delightful Waiting Experience

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 2]
```diff
@@ -8,19 +8,19 @@ export const BenefitsInfo = () => (
...
     <img src="/images/corona.png" alt="background-illustration" className={styles['corona-2']} />
     <img src="/images/corona.png" alt="background-illustration" className={styles['corona-3']} />
-    <h2 data-aos="zoom-in">Why SimplQ ?</h2>
+    <h2 data-aos="zoom-in">Your customers will love it.</h2>
     <div data-aos="zoom-in" className={`${styles['container']} ${styles['benefits-container']}`}>
       <div className={styles.benefit}>
         <img src="/images/minimize_crowding.svg" alt="minimize crowding" />
-        <p>No more waiting in long lines</p>
+        <p>Reduces waiting time and improves patient satisfaction.</p>
       </div>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:16]: Did you mean patient or customer?

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 3]
```diff
@@ -42,25 +42,45 @@ export const HowToCreate = () => (
...
               <p>
                 Manage your queues seamlessly and efficiently to provide a better user experience
               </p>
             </li>
           </ul>
         </div>
+        <p>
+          SimplQ customer waiting experience software is a simple and affordable ticket, table or
+          place in line solution, adding value to any crowded business.
+        </p>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:70]: Let's say place-in-line? Also, how about saying business involving waiting crowds?

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 4]
```diff
@@ -42,25 +42,45 @@ export const HowToCreate = () => (
...
+        <p>
+          What makes us different? Unlike other similar solutions, SimplQ is open source software
+          build by software enthusiasts from around the world. This lets us bring you the core
+          functionalities in our free plan without unnecessary fees. We also have an affordable
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:74]: built

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 5]
```diff
@@ -42,25 +42,45 @@ export const HowToCreate = () => (
...
+          month-to-month subscription option if you want to be a long term supporter. Talk to us if
+          you need dedicated support or have any custom requirements.
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:76]: long term customer?

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 6]
```diff
@@ -42,25 +42,45 @@ export const HowToCreate = () => (
...
+        </p>
+        <p>
+          We are generic enough to allow customers to choose the best way for them to use it but
+          small enough to care about the individual needs of every customer.
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:80]: it, but

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 7]
```diff
@@ -42,25 +42,45 @@ export const HowToCreate = () => (
...
+        </p>
+
+        <h3>This is what makes us stand out from other crowd control solutions.</h3>
       </div>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/StaticInfos.jsx:84]: crowd-control

File: simplq/src/components/pages/Home/LandingPage.jsx [hunk 8]
```diff
@@ -18,8 +18,8 @@ export default () => {
         <Header className={styles['main-header']}>SimplQ</Header>
         <p className={styles.subtitle}>{subtitle}</p>
         <p className={styles.description}>
-          Create and manage queues with our free online queue management solution for easy and
-          instant crowd control
+          Web based free queue management software to let your customers feel more relaxed and
+          delighted.
         </p>
```
Comment (maaverik) [inline simplq/src/components/pages/Home/LandingPage.jsx:23]: Let's just say Web based free queue management software?

File: simplq/src/components/pages/Home/StaticInfos.jsx (hunk 3 above)
Comment (daltonfury42) [reply simplq/src/components/pages/Home/StaticInfos.jsx:70]: I've made it `place-in-line`. "crowded business" seems to be a good usage to me that "business involving waiting crowds".

File: simplq/src/components/pages/Home/StaticInfos.jsx (hunk 2 above)
Comment (daltonfury42) [reply simplq/src/components/pages/Home/StaticInfos.jsx:16]: customer. Changed

File: simplq/src/components/pages/Home/StaticInfos.jsx (hunk 5 above)
Comment (daltonfury42) [reply simplq/src/components/pages/Home/StaticInfos.jsx:76]: "supporter" is nice, I thought

File: simplq/src/components/pages/Home/LandingPage.jsx (hunk 8 above)
Comment (daltonfury42) [reply simplq/src/components/pages/Home/LandingPage.jsx:23]: You mean to drop the "to let your customers feel more relaxed and delighted."? I was liking it.

**Review by maaverik — APPROVED** [review]

**Pull request merged**